node/grpc/server_v2.go

```diff
@@ -18,23 +18,26 @@ type ServerV2 struct {
 	pb.UnimplementedDispersalServer
 	pb.UnimplementedRetrievalServer
 
-	node   *node.Node
-	config *node.Config
-	logger logging.Logger
-
-	ratelimiter common.RateLimiter
-
-	mu *sync.Mutex
+	config                *node.Config
+	node                  *node.Node
+	ratelimiter           common.RateLimiter
+	assignmentCoordinator core.AssignmentCoordinator
+	logger                logging.Logger
 }
 
 // NewServerV2 creates a new Server instance with the provided parameters.
-func NewServerV2(config *node.Config, node *node.Node, logger logging.Logger, ratelimiter common.RateLimiter) *ServerV2 {
+func NewServerV2(
+	config *node.Config,
+	node *node.Node,
+	logger logging.Logger,
+	ratelimiter common.RateLimiter,
+) *ServerV2 {
 	return &ServerV2{
-		config:      config,
-		logger:      logger,
-		node:        node,
-		ratelimiter: ratelimiter,
-		mu:          &sync.Mutex{},
+		config:                config,
+		node:                  node,
+		ratelimiter:           ratelimiter,
+		assignmentCoordinator: &core.StdAssignmentCoordinator{},
```

Comment:
Why is the v1 assignment coordinator needed here?

Reply:
removed